packages/aws-cdk-lib/aws-stepfunctions-tasks/test/athena/start-query-execution.test.ts

```diff
@@ -224,4 +224,58 @@ describe('Start Query Execution', () => {
       }),
     });
   });
+
+  test('execution parameters', () => {
```

Comment:
Test cases should be added to check that validation rules are enforced.

Reply:
Okay, will add them. Should probably also add a test case to see that "token" works for example with `JsonPath.array`

Reply:
Right, understood.